pkgs/development/compilers/llvm/16/openmp/gnu-install-dirs.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
```

Comment:
That file is the same as the 15's. Why are we duplicating them?

Reply:
Because this is a WIP and we are figuring out which patch requires upgrade or not.